scripts/seed-database/upsert-production-flows.js

```diff
@@ -1,4 +1,4 @@
-import { groupBy, cloneDeep } from "lodash";
+import groupBy from "lodash/groupBy";
```

Comment:
I think this should be a two-line revert? 

```
import groupBy from "lodash/groupBy";
import cloneDeep from "lodash/cloneDeep";
```

Reply:
`cloneDeep` is an unused import 👌

Reply:
![image](https://user-images.githubusercontent.com/20502206/195360617-52c398d4-91e4-40e8-84e8-288724f35434.png)